lib/taskana-core/src/test/java/acceptance/classification/CreateClassificationAccTest.java

```diff
@@ -1,297 +1,374 @@
-    assertThat(createdClassification.getKey()).isEqualTo("Key1");
-
-    // Check 2 new created
-    long amountOfClassificationsAfter = CLASSIFICATION_SERVICE.createClassificationQuery().count();
-    assertThat(amountOfClassificationsAfter).isEqualTo(countClassificationsBefore + 2);
-
-    // Check main
-    classification = CLASSIFICATION_SERVICE.getClassification(classification.getId());
-    assertThat(classification).isNotNull();
-    assertThat(classification.getCreated()).isNotNull();
-    assertThat(classification.getModified()).isNotNull();
-    assertThat(classification.getId()).isNotNull();
-    assertThat(classification.getIsValidInDomain()).isTrue();
-    assertThat(classification.getId()).startsWith(IdGenerator.ID_PREFIX_CLASSIFICATION);
+        classificationService.newClassification("Key1", "DOMAIN_A", "TASK");
 
-    // Check master-copy
-    classification = CLASSIFICATION_SERVICE.getClassification(classification.getKey(), "");
-    assertThat(classification).isNotNull();
-    assertThat(classification.getCreated()).isNotNull();
-    assertThat(classification.getModified()).isNotNull();
-    assertThat(classification.getId()).isNotNull();
-    assertThat(classification.getIsValidInDomain()).isFalse();
-    assertThat(classification.getId()).startsWith(IdGenerator.ID_PREFIX_CLASSIFICATION);
+    classificationService.createClassification(classification);
+
+    ClassificationImpl expectedMasterClassification =
+        (ClassificationImpl) classification.copy("Key1");
+    expectedMasterClassification.setDomain("");
+
+    List<ClassificationSummary> classifications =
+        classificationService.createClassificationQuery().keyIn("Key1").list();
+
+    assertThat(classifications)
+        .usingRecursiveFieldByFieldElementComparatorIgnoringFields("id")
+        .containsExactlyInAnyOrder(
+            classification.asSummary(), expectedMasterClassification.asSummary());
   }
 
   @WithAccessId(user = "businessadmin")
   @Test
-  void testCreateClassificationWithExistingMaster() throws Exception {
-
-    Classification classification = CLASSIFICATION_SERVICE.newClassification("Key0815", "", "TASK");
-    classification.setServiceLevel("P1D");
-    CLASSIFICATION_SERVICE.createClassification(classification);
-
-    long amountOfClassificationsBefore = CLASSIFICATION_SERVICE.createClassificationQuery().count();
-    Classification expected =
-        CLASSIFICATION_SERVICE.newClassification("Key0815", "DOMAIN_B", "TASK");
-    expected.setServiceLevel("P1D");
-    Classification actual = CLASSIFICATION_SERVICE.createClassification(expected);
-    long amountOfClassificationsAfter = CLASSIFICATION_SERVICE.createClassificationQuery().count();
-
-    assertThat(amountOfClassificationsAfter).isEqualTo(amountOfClassificationsBefore + 1);
-    assertThat(actual).isSameAs(expected);
-    assertThat(actual.getIsValidInDomain()).isTrue();
+  void should_CreateClassification_WhenAlreadyExistingMasterClassification() throws Exception {
```

Comment:
Is this only a suggestion for a name change? If yes, that's not what the test does. However, we could test if no master classification is created but I think this would change the test from it's original purpose (see old version).